day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,44 @@
+10.downto(1).each do |x|
+
+  new_frog_number = x - 1
+  if new_frog_number > 1 || new_frog_number == 0
+    was_were = "were"
+  else
+    was_were = "was"
+  end
+
+  if new_frog_number > 1 || new_frog_number == 0
+    new_frog_plural = "frogs"
+  else
+    new_frog_plural = "frog"
+  end
+
```

Comment:
You can probably combine your first and second conditionals, then your 3rd ands 4th ones. You can set 2 different variables within an if block.